### PR TITLE
Add v1.1.0 upgrade doc

### PR DESCRIPTION
**Pull request by bk201**
- Arrange menu orders.
- Add v1.1.0 upgrade doc and known issues.

Previews:
- https://deploy-preview-217--harvester-preview.netlify.app/v1.1/upgrade/automatic
- https://deploy-preview-217--harvester-preview.netlify.app/v1.1/upgrade/v1-0-3-to-v1-1-0

### CODE REVIEW

**Review by vickyhella — APPROVED** [review]
Other than a few nits, LGTM. Thanks!

File: docs/upgrade/v1-0-3-to-v1-1-0.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## General information
+
+:::caution
+
+- Starting from version v1.1.0, Harvester brings in the new [VLAN enhancement](https://github.com/harvester/harvester/issues/2236) feature. Due to the implementation changes, all user VMs must shut down during an upgrade. Please stop the VMs before an upgrade.
+
+:::
+
+Once there is an upgradable version, the Harvester GUI Dashboard page will have an upgrade button. For more details, please refer to [start an upgrade](./automatic.md#start-an-upgrade).
```
[the remark this answers is not in the document]
Comment (bk201) [reply docs/upgrade/v1-0-3-to-v1-1-0.md]: Apply all suggestions, thanks @vickyhella!

**Review by guangbochen — APPROVED** [review]
LGTM and please squash the commit, thanks.

**Review by Vicente-Cheng — APPROVED** [review]
LGTM! Please squash these commits, thanks!

**Comment by bk201** [comment]
Thanks, the merged one is squashed: https://github.com/harvester/docs/commit/247ceee8e66e48e785ae488fb818465d1eff524b

**Pull request merged**